composer.json

```diff
@@ -30,6 +30,10 @@
         "symfony/filesystem": "^3.0",
         "symfony/finder": "^3.0",
         "symfony/process": "^3.0",
-        "symfony/yaml": "^3.1"
+        "symfony/yaml": "^3.1",
+        "twig/twig": "^2.0"
+    },
+    "suggest": {
+        "twig/twig": "Add support for rendering patterns using a Twig extension function."
```

Comment:
Just to be consistent with our other repos (eg https://github.com/elifesciences/api-client-php/blob/560963f8d5c61673cebe4c51c733ce19b954208c/composer.json#L36-L39) could this be `^2.0, to use PatternExtension`.

Reply:
Yep, good idea.